apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js

```diff
@@ -19,17 +19,22 @@ const {
  * @param {string} tokenType - the type of token i.e. color, spacing, radius
  * @returns {string} the SCSS variable name
  */
-const createScssTokenName = (tokenKey, tokenType) => `$${tokenType}-${tokenKey}`;
+const createScssTokenName = (tokenKey, tokenType, path = '') => {
+    const prefixExcludes = [path === 'radius.alias' ? tokenTypes.RADIUS : []];
+    const shouldShowPrefix = tokenType && !prefixExcludes.includes(tokenType);
+
+    return shouldShowPrefix ? `$${tokenType}-${tokenKey}` : `$${tokenKey}`;
+};
 
 /**
  * Creates a display name of the provided token. 'system-purple' would become 'System Purple'
  * @param {string} tokenKey - the token key i.e. 'support-positive-02'
  * @param {string} tokenType - the type of token i.e. color, spacing, radius
  * @returns {string} the display name of the token
  */
-const createTokenDisplayName = (tokenKey, tokenType) => {
+const createTokenDisplayName = (tokenKey, tokenType, path = '') => {
     // Some tokens don't require a prefix in front of their display names
-    const prefixExcludes = [tokenTypes.COLOR];
+    const prefixExcludes = [tokenTypes.COLOR, path === 'radius.alias' ? tokenTypes.RADIUS : []];
```

Comment:
same here regarding splitting logic based on token type

Reply:
📓 fyi `tokenMetadata` accepts `displayName` as a property, so if you think normalizing the token adds extra overhead, we can  just pass the value directly like [here](https://github.com/justeattakeaway/pie/blob/main/apps/pie-microsite/src/tokensMetadata.json#L643)